### PR TITLE
added release-0.5.5

### PR DESCRIPTION
**Pull request by tjsavage**

### CODE REVIEW

**Comment by robdodson** [comment]
lgtm

File: _posts/2015-02-23-release-0.5.5.markdown [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+---
+
+_We announce significant releases here on the blog, highlighting the major new features as well as breaking changes. Bear in mind, Polymer is in Developer Preview as we continue to explore the new world of Web Components, and these releases may have particularly high entropy. As always, we'd love to get your feedback, issues, and PR's, so tweet us [@polymer](https://twitter.com/polymer) or contribute to the [Github](https://github.com/Polymer)._
+
+A new, small update to Polymer, the core and paper elements, and the Web Components polyfill has been [released](https://github.com/polymer/polymer/releases).
+
+## _New_ `paper-input-decorator` character counter
+
+Awesome work from [notwaldorf](https://github.com/notwaldorf) with the brand-new character-counter addition to `paper-input-decorator`. The character counting feature matches the material design spec, and allows for custom error messages.
```
Comment (ebidel) [inline _posts/2015-02-23-release-0.5.5.markdown]: nice shout out

File: _posts/2015-02-23-release-0.5.5.markdown [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+    <paper-input-decorator label="input with 10 character max"
+         error="maximum characters exceeded">
+      <input id="i2" is="core-input" maxlength="10">
+      <paper-char-counter class="counter" target="i2"></paper-char-counter>
+    </paper-input-decorator>
+    
+<img src="/images/releases/paper-input-char-counter.png" style="height: 300px; width: auto; margin-left: auto; margin-right: auto;" alt="paper input decorator image">
+
+## _New_ `paper-tabs` now have `alignBottom` attribute
```
Comment (ebidel) [inline _posts/2015-02-23-release-0.5.5.markdown]: have -> has an

**Comment by ebidel** [comment]
LGTM. nits.

**Comment by tjsavage** [comment]
thanks for the feedback! updated

**Pull request merged**